### ISSUE TITLE
TypeError: ord() expected string of length 1, but int found

### ISSUE THREAD

**Issue by wittrup** [sviehb/jefferson#2]
Hi

After writing the script to working in python 3.5.0, I ended up with this error:

```
$ binwalk -e 100AAJX10D0.bin

DECIMAL       HEXADECIMAL     DESCRIPTION
--------------------------------------------------------------------------------
131072        0x20000         JFFS2 filesystem, big endian

wittrup@desktop MINGW64 /c/Users/wittrup/Downloads/cpe-zyxel-p8702n/_100AAJX10D0.bin.extracted
$ jefferson 20000.jffs2
Traceback (most recent call last):
  File "C:/Users/wittrup/AppData/Local/Programs/Python/Python35-32/Scripts/jefferson", line 450, in <module>
    main()
  File "C:/Users/wittrup/AppData/Local/Programs/Python/Python35-32/Scripts/jefferson", line 429, in main
    fs_list = scan_fs(content, cstruct.BIG_ENDIAN, verbose=args.verbose)
  File "C:/Users/wittrup/AppData/Local/Programs/Python/Python35-32/Scripts/jefferson", line 304, in scan_fs
    inode.unpack(content[0 + offset:])
  File "C:/Users/wittrup/AppData/Local/Programs/Python/Python35-32/Scripts/jefferson", line 208, in unpack
    self.data = rtime.decompress(node_data, self.dsize)
  File "C:\Users\wittrup\AppData\Local\Programs\Python\Python35-32\lib\site-packages\jefferson\rtime.py", line 7, in decompress
    value = ord(data_in[pos])
TypeError: ord() expected string of length 1, but int found
```

Any clues?

**Comment by wittrup** [comment]
So, I've fixed that with
`value = ord(data_in[pos]) if type(data_in) == str else data_in[pos]`
`repeat = ord(data_in[pos]) if type(data_in) == str else data_in[pos]`

Now there is another error:

```
wittrup@desktop MINGW64 /c/Users/wittrup/Downloads/cpe-zyxel-p8702n/_100AAJX10D0.bin.extracted
$ jefferson 20000.jffs2
data length mismatch!
Traceback (most recent call last):
  File "C:/Users/wittrup/AppData/Local/Programs/Python/Python35-32/Scripts/jefferson", line 450, in <module>
    main()
  File "C:/Users/wittrup/AppData/Local/Programs/Python/Python35-32/Scripts/jefferson", line 430, in main
    fs_list += scan_fs(content, cstruct.LITTLE_ENDIAN, verbose=args.verbose)
TypeError: unsupported operand type(s) for +=: 'dict_values' and 'dict_values'
```

**Comment by wittrup** [comment]
Issue fixed with c6211c438088c951fcfab2321eac4abac9047823
